lib/plugins/inspec-init/lib/inspec-init/cli_plugin.rb

```diff
@@ -0,0 +1,242 @@
+# encoding: utf-8
+
+require_relative 'renderer'
+
+module InspecPlugins
+  module Init
+    class CLI < Inspec.plugin(2, :cli_command)
+      #-------------------------------------------------------------------#
+      #                 inspec init plugin
+      #-------------------------------------------------------------------#
+      desc 'PLUGIN_NAME [options]', 'Generates an InSpec plugin, which can extend the functionality of InSpec itself.'
+      # General options
+      option :prompt, type: :boolean, default: true, desc: 'Interactively prompt for information to put in your generated plugin.'
+      option :detail, type: :string, default: 'full', desc: "How detailed of a plugin to generate. 'full' is a normal full gem with tests; 'core' has tests but no gemspec; 'test-fixture' is stripped down for a test fixture."
+
+      # Templating vars
+      option :author_email, type: :string, default: 'you@example.com', desc: 'Author Email for gemspec'
+      option :author_name, type: :string, default: 'Your Name', desc: 'Author Name for gemspec'
+      option :description, type: :string, default: '', desc: 'Multi-line description of the plugin'
+      option :summary, type: :string, default: 'A plugin with a default summary', desc: 'One-line summary of your plugin'
+      option :license_name, type: :string, default: 'Apache-2.0', desc: 'The name of a license'
```

Comment:
When the user provides `--license-name` we don't validate it against the licenses we have text for. That will let them supply their own license, but if they typo the license name they won't necessarily know that their generated LICENSE file contains a friendly message telling them to supply text. 

Could we emit something to the UI indicating that the license $name was not found, and they'd need to update license.md with their own license?

Reply:
OK, I've added a reminder if the user chooses Other:

![screen shot 2019-02-06 at 12 10 14](https://user-images.githubusercontent.com/156460/52359611-3fccc480-2a08-11e9-82a6-fb2cceb31e3c.png)
![screen shot 2019-02-06 at 12 10 24](https://user-images.githubusercontent.com/156460/52359619-43604b80-2a08-11e9-8bf1-7ac8d20b6812.png)